src/storage/tiff.cpp

```diff
@@ -458,6 +466,27 @@ Tiff::get(struct StorageProperties* settings) const noexcept
     settings->pixel_scale_um = pixel_scale_um_;
 }
 
+void
+Tiff::get_meta(struct StoragePropertyMetadata* meta) noexcept
+{
+    (*meta) = {
+        .file_control = { .supported = 1, .default_extension = { 0 } },
+        .external_metadata = { .writable = 1, .type = PropertyType_String },
+        .first_frame_id = { 0 },
+        .pixel_scale = { .x = { .writable = 1,
+                                .low = 0.0f,
+                                .high = std::numeric_limits<float>::infinity(), // TODO (aliddell: what's best here?)
```

Comment:
If you really want, this could be calculated based on `std::numeric_limits` and the rational precision/calculation done in the TIFF implementation (e.g. multiplying by 10000 to go from um -> cm to get the denominator). Not my idea of a fun time, but doable. I also wonder how the floating point -> rational conversion handles overflow.